### PR TITLE
Various small clean-up changes accumulated during the last release.

### PR DESCRIPTION
**Pull request by afs**

### CODE REVIEW

**Review by rvesse — APPROVED** [review]

File: jena-arq/src/main/java/org/apache/jena/http/auth/AuthEnv.java [hunk 1]
```diff
@@ -59,6 +59,11 @@ private AuthEnv() { }
     public void registerUsernamePassword(URI uri, String user, String password) {
         AuthDomain domain = new AuthDomain(uri);
         passwordRegistry.put(domain, new PasswordRecord(user, password));
+        // Remove any existing registration for this URI,
+        // but not registrations with this URI as prefix.
+        // unregisterUsernamePassword() removes registration
+		// with the URI as prefix.
```
Comment (kinow) [inline jena-arq/src/main/java/org/apache/jena/http/auth/AuthEnv.java:65]: tabs?

File: jena-cmds/src/test/java/arq/rdftests.java [hunk 2]
```diff
@@ -86,25 +86,32 @@ public static void main(String...argv) {
...
+
     protected rdftests(String[] argv) {
         super(argv);
-        super.add(strictDecl, "--strict", "Operate in strict mode (no extensions of any kind)");
-        super.add(baseDecl, "--base=URI", "Set the base URI");
+//        super.add(baseDecl, "--base=URI", "Set the base URI");
         super.modVersion.addClass(ARQ.class);
         getUsage().startCategory("Tests (execute test manifest)");
         getUsage().addUsage("<manifest>", "run the tests specified in the given manifest");
+        add(arqDecl, "--arq",       "Operate with ARQ syntax)");
```
Comment (kinow) [inline jena-cmds/src/test/java/arq/rdftests.java:113]: Extra `)` after syntax?

File: jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/server/Dispatcher.java [hunk 3]
```diff
@@ -58,22 +58,40 @@ public class Dispatcher {
...
-     *
+     * <p>
      * This function does not throw exceptions.
+     * <p>
+     * The dispatch process is:
+     * <ul>
+     * <li> Decide the data, based on the request URI ({@link #locateDataAccessPoint(HttpServletRequest, DataAccessPointRegistry)}
+     * <li> Allocate HttpAction ({@link #process}, {@link #dispatchAction}).
+     * <li> Decide service endpoint name ({@link #chooseProcessor})
+     * <li> Decvice operation {@link #chooseEndpoint}
```
Comment (kinow) [inline jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/server/Dispatcher.java:80]: s/Decvice/Decide (also not sure if the space between `<li>` and text matter, probably it doesn't)

File: jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/server/Dispatcher.java [hunk 4]
```diff
@@ -58,22 +58,40 @@ public class Dispatcher {
...
+     *   <li> Content type  ({@link #chooseOperation})
+     *   <li> Default - quads operation
+     *   </ul>
+     * <li>Allow authentication for the dispatch choice.
+     * </ul>
+     *
      */
     public static boolean dispatch(HttpServletRequest request, HttpServletResponse response) {
         DataAccessPointRegistry registry = DataAccessPointRegistry.get(request.getServletContext());
+        // Use the name to choose a DataAccessPoint. This is dispatch to the right dataset.
```
Comment (kinow) [inline jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/server/Dispatcher.java:92]: dispatched?

**Review by kinow — APPROVED** [review]

**Pull request merged**